node/src/chain_spec.rs

```diff
@@ -52,7 +52,7 @@ fn get_session_keys(
 	im_online: ImOnlineId,
 	authority_discovery: AuthorityDiscoveryId,
 	) -> SessionKeys {
-	SessionKeys { grandpa, babe, im_online, authority_discovery }
+	SessionKeys { babe, grandpa, im_online, authority_discovery }
```

Comment:
This looks like a bug in genesis. Probably safe to merge, but need to double check.